cves/kernel/CVE-2013-0290.yml

```diff
@@ -473,5 +508,5 @@ nickname_instructions: |
   A catchy name for this vulnerability that would draw attention it.
```

Comment:
Can't comment unchanged code, so putting this below it:

CWE_note is not filled out. Make sure you look over CWE 20 and confirm it.